### ISSUE TITLE
On a ranklens dataset, user-metadata events do not affect the ranking.

### ISSUE THREAD

**Issue by shuttie** [metarank/metarank#473]
Reported in slack, original message:
> In the demo, in the events data on which it’s being trained (events.jsonl.gz), we don’t see user metadata being included or user features being a part of the config file. We tried creating our own sample events for training and while the bootstrapping model accepted our ‘ranking’, ‘interaction’ and ‘metadata-item’ events, it didn’t work when we tried to include ‘metadata-user’ event (similar to ‘metadata-item’ event but with ‘user’ instead of ‘item’). Is it currently not possible to include user features in the xgboost model ?

It should be possible to use user features per-se, but our test coverage is not good related to the user features.

**Comment by GautamR-Samagra** [comment]
Our Sample code for creating sample user metadata (python): 

```
no_of_users =  10

for id1 in range(no_of_users):
   
    metadata_users_dict.append({
        "event": "metadata",
        "fields": [{
                "name": "land_area",
                "value": random.randint(100,250)/100.00
                },
            {
                "name": "crops",
                "value": random.choice(['rabi','kharif'])
                }],
        "id": str(id1 + 1),
        "user": "user"+ str(id1+1),
        "tenant": "default",
        "timestamp":  100 + id1     
    })

metadata_users_dict[0:2]
```

**Comment by GautamR-Samagra** [comment]
This got fixed when I used the correct event type e.g:  

```
{
  "event": "user", 
  "id": "1", 
  "timestamp": 100,
   "user": "user1", 
  "fields": [{"name": "land_area", "value": 2.19},
   {"name": "crops", "value": "kharif"}]
}

```

**Comment by shuttie** [comment]
fixed in 0.5.x
